.github/workflows/on-merge-queue.yml

```diff
@@ -0,0 +1,49 @@
+---
+# Merge queue.  A PR that reaches here has already passed the more rigorous PR
+# suite, so is very likely to pass.  The main reasons for failures here are
+# flaky VMs timing out (which we can't do much about), or a merge conflict
+# with another PR that is also in the merge queue.
+#
+# There's no reason to have multiple stages in this case, because we're
+# expecting it to pass.  Having more than one stage frustrates parallel
+# throughput in low-contention cases, and guarantees a longer critical path.
+name: Merge queue
+on:
+  # remove when proven to work
+  pull_request:
```

Comment:
Should we mark this as on hold until you remove this?

Reply:
Agreed, so that I don't randomly forget :)